astropy/analytic_functions/mars_orbital.py

```diff
@@ -0,0 +1,11 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Functions related to orbital calculations for Mars."""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+__all__ = ['plug_my_laptop_in']
```

Comment:
I suggest to rename the function to ``plug_my_laptop_in_Pleiades`` in order to remove any ambiguity.